### PR TITLE
Update labels, names, and concordances in region records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/1640.

This PR:

- Updates English `name` and `label` properties in region records
- Adds missing Indonesian `label` properties in region records
- Adds missing the Wikipedia concordances in region records
- Updates population, population date, and population rank properties with. newer population data

No PIP work required, can merge once approved.

### CODE REVIEW

File: data/856/719/73/85671973.geojson [hunk 1]
```diff
@@ -577,14 +587,14 @@
         "ind",
         "eng"
     ],
-    "wof:lastmodified":1617130624,
+    "wof:lastmodified":1623109425,
     "wof:name":"Aceh",
     "wof:parent_id":85632203,
     "wof:placetype":"region",
     "wof:placetype_alt":[],
     "wof:placetype_local":"autonomous province",
```
Comment (nvkelso) [inline data/856/719/73/85671973.geojson]: Should this value also change to be consistent with the preferred placetype change above?

File: data/856/720/45/85672045.geojson [hunk 2]
```diff
@@ -309,8 +318,9 @@
         "quattroshapes"
     ],
     "src:lbl:centroid":"mapshaper",
-    "src:population":"statoids",
+    "src:population":"wk",
     "src:population:date":"2010-05-17",
+    "src:population_date":"2020-01-01",
```
Comment (nvkelso) [inline data/856/720/45/85672045.geojson:322]: > "src:population:date":"2010-05-17",
>  "src:population_date":"2020-01-01",

Only one of these should be true... so delete `src:population:date` from this PR for this and other relevant records please.

**Review by nvkelso — CHANGES_REQUESTED** [review]
Couple nits

File: data/856/719/73/85671973.geojson (hunk 1 above)
Comment (stepps00) [reply data/856/719/73/85671973.geojson]: Yeah, good catch.. I'll double check the `wof:placetype_local` properties in this PR.

File: data/856/720/45/85672045.geojson (hunk 2 above)
Comment (stepps00) [reply data/856/720/45/85672045.geojson:322]: Will take care of this in an upcoming commit...

**Comment by stepps00** [comment]
The most recent commits removes the bogus `src:population:date` properties and corrects `wof:placetype_alt` properties in a handful of records. This PR is ready for another review.

**Review by nvkelso — APPROVED** [review]
Thanks for the changes!

**Pull request merged**